### PR TITLE
Intake Revisions and CAN with CANivore fixed

### PR DESCRIPTION
**Pull request by GucciBanana11**
Stuff we worked on with Justin on Monday

### CODE REVIEW

**Review by centiwati — APPROVED** [review]

File: src/main/java/frc/robot/subsystems/Shooter.java [hunk 1]
```diff
@@ -39,6 +40,7 @@ public void shooterInit() {
     SmartDashboard.putNumber(SMART_DASHBOARD_TARGET_VELOCITY, targetVelocity);
     SmartDashboard.putString(SMART_DASHBOARD_POSITION, positionOfArm.toString());
     leftShootMotor.setInverted(true); // TODO: Change to Right/Left to invert shooting motor
+    TalonFX.optimizeBusUtilizationForAll(pivotMotor, leftShootMotor, rightShootMotor);
```
Comment (Jnesselr) [inline src/main/java/frc/robot/subsystems/Shooter.java:43]: I think you have to make sure to enable the ones you do care about before calling this method. Unsure if that's why the motors aren't working or not.

File: vendordeps/Phoenix6.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
     "fileName": "Phoenix6.json",
     "name": "CTRE-Phoenix (v6)",
-    "version": "24.1.0",
+    "version": "24.2.0",
```
Comment (Jnesselr) [inline vendordeps/Phoenix6.json:4]: Next time y'all update a library, do that in its own PR

**Review by Jnesselr — APPROVED** [review]
Approving to enable other work to continue. The intake state machine still needs some love.
